public_suffix_list.dat

```diff
@@ -13809,4 +13813,6 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+
```

Comment:
Please remove these two extra blank lines at bottom - there are some that integrate the PSL downstream that this has caused problems for in the past.

Reply:
@dnsguru Please forgive me for these :)  I think I did it correctly now.